services/libs/data-access-layer/src/organizations/organizations.ts

```diff
@@ -308,6 +308,25 @@ export async function findMemberOrganizations(
   )
 }
 
+export async function findOrganizationsByIds(
+  qe: QueryExecutor,
+  ids: string[],
+  fields = '*',
+): Promise<IMemberOrganization[]> {
+  const formattedIds = ids.join(', ')
+  return await qe.select(
+    `
+    SELECT ${fields}
+    FROM "organizations"
+    WHERE id IN ($(ids))
+    `,
+    {
+      ids: formattedIds,
+      fields,
+    },
+  )
+}
+
```

Comment:
And this we already have `queryOrgs()` in `services/libs/data-access-layer/src/orgs/index.ts`

Used like this:
```
          await queryOrgs(qx, {
            filter: {
              [OrganizationField.ID]: {
                in: orgIds,
              },
            },
            fields: [OrganizationField.ID, OrganizationField.DISPLAY_NAME, OrganizationField.LOGO],
          })
```